packages/eslint-plugin-office-addins/src/utils/utils.ts

```diff
@@ -72,12 +82,28 @@ function findOfficeApiReferencesInScope(scope: Scope): void {
   scope.childScopes.forEach(findOfficeApiReferencesInScope);
 }
 
+function isCallingMethod(node: TSESTree.Node | undefined): boolean {
```

Comment:
Should do like the 'isLoadReference' method and pass in the identifier type and worry about the undefined in the next if statement

Reply:
Changed it to not have to worry about identifier and also not needs to send `node.parent`. You now just pass `node` which makes it easier

Reply:
That seems to be the way we've been coding (see isLoadFunction) . . . we've opted for more strongly typed function arguments.  It does look like that pushes us to determine type sooner rather then later and possibly in more places, but I can't remember specifically why we ended up going that way (I do vaguely remember trying in out in each way).  We should be uniform so you could change the other calls to do it this way or you could change this to match the others.